.gitmodules

```diff
@@ -10,3 +10,6 @@
 [submodule "firecracker"]
 	path = firecracker
 	url = https://github.com/firecracker-microvm/firecracker.git
+[submodule "tests/perf/s2n-quic"]
+	path = tests/perf/s2n-quic
+	url = https://github.com/zhassan-aws/s2n-quic
```

Comment:
Why not s2n-quic's repo?

Reply:
We generally need to make changes to integrate with our regressions:
1. Adding Kani harnesses (this may not be needed if we're able to run the Bolero harnesses directly using Kani)
2. Adding expected files so that they work with compiletest 
Since we can't push changes to s2n-quic, I created a fork that we can push those changes to.